### DISCUSSION TITLE
Add some sensors

### DISCUSSION THREAD

**Discussion by tlamoureux24** [hassio-addons/addon-ssh#496] [Ideas]
Hello,

Is it possible to add some sensors like in the official addon ?
Some community addons have these sensors too (phpmyadmin, log viewer, zwave js ui for example)

It could be  :
binary_sensor.ssh_web_terminal_running
binary_sensor.ssh_web_terminal_update_available
sensor.ssh_web_terminal_cpu_percent
sensor.ssh_web_terminal_memory_percent
sensor.ssh_web_terminal_newest_version
sensor.ssh_web_terminal_version

Thank you.

**Comment by frenck** [comment]
That is not an add-on concern or provided by add-ons. Integrations create entities. Add-ons are not integrations.

../Frenck

**Comment by tlamoureux24** [comment]
It looks clear for you but for the entire earth it's not, sorry.
For example with phpmyadmin, log viewer and zwave js ui the only thing i do is install these community addons and the sensors were here ......
But ok, sorry to have disturbed.

**Reply by frenck** [reply]
> It looks clear for you but for the entire earth it's not, sorry.

Understood, hence my comment above explaining it 🤷
